app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less

```diff
@@ -45,6 +45,17 @@
         }
     }
 
+    .products-grid.wishlist {
+        .product-item {
```

Comment:
Can these styles be moved to line 137 (.products-grid.wishlist { .product { &-item)?

Reply:
I have applied changes as per your suggestion